pkg/apis/gcp/validation/shoot_test.go

```diff
@@ -206,6 +206,65 @@ var _ = Describe("Shoot validation", func() {
 			))
 		})
 
+		It("should forbid because service account's email is empty", func() {
```

Comment:
Can you rather move these tests to `worker_test.go` instead of `shoot_test.go` as your validation code belongs to implementation in `worker.go`?